lib/report-translator.js

```diff
@@ -121,7 +137,7 @@ function compareFixesByRange(a, b) {
  * Merges the given fixes array into one.
  * @param {Fix[]} fixes The fixes to merge.
  * @param {SourceCode} sourceCode The source code object to get the text between fixes.
- * @returns {{text: string, range: [number, number]}} The merged fixes
+ * @returns {{text: string, range: number[]}} The merged fixes
```

Comment:
Not familiar with JSDoc. Can we do something like `number[2]` or otherwise represent the ranges as a 2-tuple somehow?

Reply:
Unfortunately it doesn't work, and I haven't found any details about specifying the length of the array on the internet. I don't think JSDoc supports it.